### PR TITLE
LRDOCS-10256 Give more in-depth description of color selection when editing Style Book

### PR DESCRIPTION
**Pull request by Alec-Shay**
See: https://issues.liferay.com/browse/LRDOCS-10256

These changes add a new section explaining the interface for selecting/customizing colors when editing a Style Book, including reusing Style Book token values.

### CODE REVIEW

File: docs/dxp/latest/en/site-building/site-appearance/style-books/using-a-style-book-to-standardize-site-appearance.md [hunk 1]
```diff
@@ -40,12 +40,42 @@ The page is published using the new Style Book.
...
 
+### Customizing Your Style Book's Colors
+
+When you are choosing a color for any color field in your Style Book, you can choose the color in multiple ways:
+
+* Click the color on the left side to open a color picker, and choose any color.
+
+* Enter the hex code for the desired color into the text box to change to any color.
+
+* Click the *Value from Stylebook* button to open a menu of pre-defined colors to choose from (including the Style Book token values, under the "Color System" category).
```
Comment (veroglez) [inline docs/dxp/latest/en/site-building/site-appearance/style-books/using-a-style-book-to-standardize-site-appearance.md:63]: When you comment _(including the Style Book token values, in the "Color System" category)_
Why is only the Color System category specified? really all the categories appear: General, Typography, Buttons, etc...

**Comment by veroglez** [comment]
Hi @Alec-Shay! It looks great! 
I just left a comment. A question that arises is, Is it possible for images that have been changed to be updated with images showing the new input? I think this would avoid confusion for the user seeing that all images where there are inputs are updated.

File: docs/dxp/latest/en/site-building/site-appearance/style-books/using-a-style-book-to-standardize-site-appearance.md (hunk 1 above)
Comment (Alec-Shay) [reply docs/dxp/latest/en/site-building/site-appearance/style-books/using-a-style-book-to-standardize-site-appearance.md:63]: Ahhh -- I see I had a misunderstanding when I tested this. I thought that the Color System category was the only category that corresponded to token values, and that you could attach and detach other options to. I re-tested and realizes that this also applies to colors from any category. So I'm pushing an update removing this comment, as well as a note that I had added that says that.

**Comment by Alec-Shay** [comment]
Thanks for the feedback, @veroglez. I pushed a couple of updates to this, to update more screenshots, and to address your other comment. Can you also review and check whether these updates address your concerns, to be sure? Thank you!

**Comment by veroglez** [comment]
I think the content is better understood with the updates. It looks great! Thanks so much @Alec-Shay!

**Comment by Alec-Shay** [comment]
Thanks for taking a look, and the feedback, @veroglez! I'll go ahead and push this forward then.

**Pull request closed without merging**